azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/AuthHelper.java

```diff
@@ -56,11 +56,11 @@ public static void setupAzureEnvironment(AzureEnvironment env) {
         }
     }
 
-    public static void setupAuthProxy(String httpProxyHost, String httpProxyPort) {
+    public static void setupAuthProxy() {
+        final String httpProxyHost = ProxyManager.getInstance().getHttpProxyHost();
```

Comment:
Can we `setupAuthProxy` in ProxyManager?

Reply:
ProxyManager in common-lib has no way to reference class in 'azure identity'